packages/bvaughn-architecture-demo/components/inspector/PropertiesRenderer.tsx

```diff
@@ -32,14 +32,16 @@ export default function PropertiesRenderer({
 }) {
   const client = useContext(ReplayClientContext);
 
-  const { className, objectId, preview } = object;
+  const { className, objectId } = object;
 
   // If we have an ObjectPreview already, use it.
   // If we just have an Object, then Suspend while we fetch preview data.
-  if (preview == null || preview.overflow) {
+  if (object.preview == null || object.preview.overflow) {
     object = getObjectWithPreview(client, pauseId, objectId, true);
   }
 
+  const { preview } = object;
```

Comment:
It's very annoying that I can't comment on lines that didn't change - but just to make sure I am following here, we had a situation where line 51 was basically referencing a stale (or potentially missing) `preview`, is that right?

Reply:
Yup ☹️